app/helpers/application_helper.rb

```diff
@@ -34,25 +34,17 @@ def currency(display_currency)
     number_to_currency(display_currency)
   end
 
-  def format_downtime(downtime)
-    downtime = downtime.to_i
-    hours = downtime / 3600
-    minutes = (downtime % 3600) / 60
-    return ("%d" % hours) + ":" + ("%02d" % minutes)
-  end
-
-  def format_remaining_downtime(downtime)
-    downtime = downtime.to_i
-    if downtime < 0
-      neg = '-'
-      downtime *= -1
+  def format_duration(time)
+    time = time.to_i
+    if time < 0
+      neg = '&minus;'.html_safe
```

Comment:
@timparenti For my own knowledge, what's the difference between '-' and the escaped character '&minus;'?

Reply:
It's just an actual minus sign where appropriate, that's all.